code/game/objects/items/kitchen.dm

```diff
@@ -96,6 +96,23 @@
 	wound_bonus = -5
 	bare_wound_bonus = 10
 	tool_behaviour = TOOL_KNIFE
+	var/armor_pene_stab = 20
+	var/force_stabby = 5
+	var/wound_bonus_stabby = 5
+
+/obj/item/kitchen/knife/attack_alt(mob/living/victim, mob/living/user, params)
+	force -= force_stabby
+	wound_bonus += wound_bonus_stabby
+	armour_penetration += armor_pene_stab
+	attack_verb_continuous = list("stabs")
+	attack_verb_simple = list("stab")
+	sharpness = SHARP_POINTY
+	attack(victim, user, params)
+	armour_penetration = initial(armour_penetration)
+	attack_verb_continuous = initial(attack_verb_continuous)
+	attack_verb_simple = initial(attack_verb_simple)
+	sharpness = initial(sharpness)
```

Comment:
I'm not sure how one would go about cleaning this up, I'd just like to say I hate it to my very core

Reply:
I hate it too

Reply:
Wont `initial` on values here that are lists evaluate to null?